services/filter_file.rb

```diff
@@ -0,0 +1,58 @@
+class FilterFile
+  def call(wait_time = 5)
+    puts I18n.t(:enter_path)
```

Comment:
You can write helper method, use Module for it
```ruby
module Translation
  def self.translate(path)
    puts I18n.t(:path)
  end
end

# then include it where it's needed

```

Reply:
Created helper method